client/src/styles/PropertiesSummary.scss

```diff
@@ -51,6 +50,39 @@
           margin-left: 0;
         }
       }
+      .warning-banner {
+        position: absolute;
+        z-index: 1000;
+        // rounding error causes bottom=0 to be off by 1px at some zoom levels,
+        // this hack seems to fix it https://stackoverflow.com/a/66623955/7051239
+        bottom: -0.4px;
+        background-color: #ffba33;
```

Comment:
There should be a new `$wowza-warning` color variable you can use here, instead!